src/lib/_imports/trumps/s-header.css

```diff
@@ -124,6 +154,12 @@ Styleguide Trumps.Scopes.Header
   padding-left: 14px; /* NO-R/EM: .875rem (overwrite Bootstrap) */
 }
 
+@media only screen and (min-width: 992px) {
+  .s-header[class*="navbar-expand-"] .navbar-nav .nav-link {
+      padding-block: 0; /* overwrite core-styles.header.css */
+  }
+}
```

Comment:
If you actually want to make this change, go for it. Otherwise, just add this comment.

And either way, please **delete** `/* overwrite core-styles.header.css */` (because that comment was relevant only when the style was in tup-ui).

```suggestion
/* TODO: Instead of removing vert. padding on wide screen,
         only add vert. padding on narrow screen */
@media only screen and (min-width: 992px) {
  .s-header[class*="navbar-expand-"] .navbar-nav .nav-link {
      padding-block: 0;
  }
}
```

Reply:
In other words:

Move [`padding-top` and `-bottom` (i.e. `padding-block`) values that have been for every screen](https://github.com/TACC/Core-Styles/blob/c22968d2582a3e027bc68de4b65fd9660b7b7f64/src/lib/_imports/trumps/s-header.css#L150-L151) to be within a new media query that only applies to relevant screens e.g.

```
@media only screen and (max-width: 991px) {
  .s-header[class*="navbar-expand-"] .navbar-nav .nav-link {
      padding-block: 8px;
  }
}
```

Reply:
<img width="2337" alt="Screenshot 2023-11-03 at 5 33 40 PM" src="https://github.com/TACC/Core-Styles/assets/63771558/575620d9-6154-4902-bd08-cd8d9f0b6ed1">